### DISCUSSION TITLE
Can I manage logs by using longhorn driver and Where longhorn storage all data?

### DISCUSSION THREAD

**Discussion by cengizhanR** [longhorn/longhorn#8749] [General]
Hello, I want to use Longhorn for persistent storage resources instead of NFS servers. However, I didnt understand where longhorn storege all data ? Does it need volume? I need to attach some files in running container images. However is it possible to attach data to container which are runnig on k8s clusters by Longhorn. Should I manage logs by using longhorn driver ? Is ıt useful.

**Comment by derekbit** [comment]
Hello @cengizhanR 
Thanks for your attention to Longhorn. Please check the longhorn official document https://longhorn.io, and it will help you understand how longhorn works.

**Reply by cengizhanR** [reply]
Hello thank you for your return @derekbit , I have just checked it and I realize that Longhorn abstracts the file management, so the actual file (myfile.txt) created in the pod is managed within these block storage files. Directly accessing and interpreting these block files isn't straightforward and typically requires Longhorn's management tools.But, How can I see my myfile.txt which is created on pod on host nodes?
